modules/apps/commerce/commerce-pricing-web/src/main/resources/META-INF/resources/js/AddPriceEntryForm.js

```diff
@@ -181,6 +187,9 @@ function AddPriceEntryForm({
 									name={`${namespace}commercePriceList${index}`}
 									onChange={({target}) => {
 										onChangePriceListHandler({
+											currency:
+												priceLists[target.selectedIndex]
```

Comment:
because this depends a lot of the order can we change it like this?
```
priceLists.find((item) => item.id === target.value).currency
```

(I don't know if it's `id` or `priceListId`)